src/main/java/cc/polyfrost/oneconfig/internal/config/core/ConfigCore.java

```diff
@@ -39,6 +39,7 @@
 
 public class ConfigCore {
     public static List<Mod> mods = new ArrayList<>();
+    public static List<Mod> subMods = new ArrayList<>();
```

Comment:
use a hashmap

Reply:
Mod and what

Reply:
mod to the sub mods

Reply:
public static HashMap<Mod, List<Mod>> subMods = new HashMap<>();

Reply:
sure just dont store them in a centralized list like that. you are using it like a hashmap anyway